addons/OPT/gps/fn_clientInit.sqf

```diff
@@ -0,0 +1,349 @@
+/**
+* Spielerdarstellung auf der Karte 
+* 
+* Autor: [GNC]Lord-MDB
+*
+* Argumente:
+* kein
+*
+* Rückgabewert:
+* kein
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+*
+* API:
+* Nein
+* 
+* Beispiel Externer Aufruf:
+* [] call EFUNC(gps,clientInit);
+* Beispiel interner Aufruf:
+* [] call FUNC(clientInit);
+*
+*/
+
+#include "macros.hpp"
+
+DUMP("Successfully loaded the OPT/GPS module on the client");
+
+// Event Liste
+#include "\opt\opt\addons\opt\tracker\Events.hpp"
+
+// Textur Liste
+#include "\opt\opt\addons\opt\tracker\Texturen.hpp"
+
+// Kontrolle ob Map oder Dialog geöffnet wird
+DFUNC(dialogCheck) = 
```

Comment:
Wird diese Funktion genutzt? Im TrackerModul werfen wir doch Events, wenn ein Display geöffnet wird. Nach meinem Verständnis wird "_verarbeitungEin" auch später im RADAR Modul gebraucht, bevor wir den Code duplizieren könnte man ihn in der TrackerAPI verwalten.

Reply:
Siehe auch: https://github.com/OperationPandoraTrigger/OPT-Mod/pull/18#discussion_r288480125 und https://github.com/OperationPandoraTrigger/OPT-Mod/pull/18#discussion_r288482547